packages/cli/doc-core/src/node/mdx/options.ts

```diff
@@ -4,19 +4,19 @@ import rehypeAutoLink from 'rehype-autolink-headings';
 import { Options } from '@mdx-js/loader';
 import { UserConfig } from 'shared/types/index';
 import { PluggableList } from 'unified';
-import remarkPluginFrontMatter from 'remark-frontmatter';
```

Comment:
can we remove the `remark-frontmatter` and `remark-mdx-frontmatter` from package.json?